xacro/__init__.py

```diff
@@ -1009,6 +1009,26 @@ def process_file(input_file_name, **kwargs):
 
     return doc
 
+def to_urdf(xacro_path, urdf_path=None):
```

Comment:
Why should we have this additional function? Shouldn't `ros_launch` call xacro as usual?